recipes/xpack/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class XpackConan(ConanFile):
+    name = "xpack"
+    description = "C++ reflection, ability to convert between C++ structures and json/xml."
+    topics = ("xpack", "json", "reflection", "xml")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xyz347/xpack"
+    license = "MIT"
```

Comment:
```suggestion
    license = "Apache-2.0"
```

The correct license is Apache 2.0 ! 

https://github.com/xyz347/xpack/blob/v1.0.2/LICENSE